tempesta_fw/t/functional/very_many_backends/test_deadtime_1M.py

```diff
@@ -0,0 +1,333 @@
+            self.recieved_chain = deproxy.MessageChain.empty()
+            self.client.clear()
+            self.client.set_request(self.current_chain.request)
+            self.loop(timeout)
+
+    def run(self, finish_event):
+        num_req = 0
+        long_times = 0
+        short_times = 0
+        max_delay = 0
+        min_delay = -1
+        start_time = time.time()
+        success_time = start_time
+        curtime = start_time
+        self.client.start()
+        while (curtime - start_time < self.timeout or self.timeout < 0) and \
+              not finish_event.is_set():
+            self.request(self.timeout)
+            curtime = time.time()
+            delay = curtime - success_time
+            if delay > self.max_deadtime:
+                long_times += 1
+            else:
+                short_times += 1
+            if delay > max_delay:
+                max_delay = delay
+            if delay < min_delay or min_delay == -1:
+                min_delay = delay
+            success_time = curtime
+            num_req += 1
+            if num_req % 10000 == 0:
+                tf_cfg.dbg(3, "%i requests done" % num_req)
+
+        delay = curtime - success_time
+        if delay > self.max_deadtime:
+            long_times += 1
+        if delay > max_delay:
+                max_delay = delay
+        tf_cfg.dbg(3, "short times: %i" % short_times)
+        tf_cfg.dbg(3, "long times: %i" % long_times)
+        tf_cfg.dbg(3, "max delay: %f" % max_delay)
+        tf_cfg.dbg(3, "min delay: %f" % min_delay)
+        if finish_event.is_set():
+            tf_cfg.dbg(3, "Finish event recieved")
+        else:
+            tf_cfg.dbg(3, "No finish event recieved")
+        self.client.stop()
+        sys.exit(long_times)
+
+class DontModifyBackend(stress.StressTest):
+    """ 1 backend in server group """
+    num_attempts = 10
+    max_deadtime = 2
+    num_extra_interfaces = 8
+    num_extra_ports = 32
+    ips = []
+    normal_servers = 0
+    config = "cache 0;\n"
+    base_port = 16384
+    wait = True
+    client = None
+    configurator = None
+
+    def setUp(self):
+        tf_cfg.dbg(2, "Creating interfaces")
+        self.interface = tf_cfg.cfg.get('Server', 'aliases_interface')
+        self.base_ip = tf_cfg.cfg.get('Server',   'aliases_base_ip')
+        self.ips = multi_backend.create_interfaces(self.interface,
+                           self.base_ip, self.num_extra_interfaces + 1)
+        stress.StressTest.setUp(self)
+
+    def tearDown(self):
+        has_base_excpt = False
+        # stopping client
+        if self.client != None:
+            self.client.stop()
+        # stopping tempesta and servers
+        try:
+            stress.StressTest.tearDown(self)
+        except Exception as exc:
+            has_base_excpt = True
+            excpt = exc
+        tf_cfg.dbg(2, "Removing interfaces")
+        for ip in self.ips:
+            multi_backend.remove_interface(self.interface, ip)
```

Comment:
Can throw an exception, the rest of cleanup won't be done.

Reply:
No, multi_backend.remove_interface don't throw any exceptions